src/helpers.ts

```diff
@@ -30,11 +30,15 @@ export function rankResolver(input: string): boolean | string {
 	return false
 }
 
-export function sortPlayers(a: Signup, b: Signup, lobby: Lobby) {
+export function sortPlayers(a: Signup, b: Signup, { region, rank }: Lobby) {
 	if (a.gamesPlayed > b.gamesPlayed) return 1
 	if (a.gamesPlayed < b.gamesPlayed) return -1
-	if (a.region === lobby.region && b.region !== lobby.region) return -1
-	if (b.region === lobby.region && a.region !== lobby.region) return 1
+	if (a.region === region && b.region !== region) return -1
+	if (b.region === region && a.region !== region) return 1
+	const aRolesInRank = Object.values(a).filter(v => v === rank).length
+	const bRolesInRank = Object.values(b).filter(v => v === rank).length
+	if (aRolesInRank > bRolesInRank) return -1
+	if (aRolesInRank < bRolesInRank) return 1
```

Comment:
So based on this I'm reading the ordering is now gamesPlayed, region, RolesInRank - should region not be first to avoid ping diffs?

Reply:
It was like this before. Region should more be like a preference, and would be pretty uncool if all streamers of a night are NA